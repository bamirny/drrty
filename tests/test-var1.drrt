function int letter(){
  int a;
  a = 18;
  return a;
}

function int main(){
  print(letter());
  makeHeader("Hello!");
  makeText("DRRTY's first program!");
  return 0;
}


