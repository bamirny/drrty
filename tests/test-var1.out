18<br>
<h1>Hello!</h1>
<p>DRRTY's first program!</p>
